services/langyagent/internal/assets/skills/evaluations/SKILL.md

```
@@ -0,0 +1,254 @@
+
+await evaluation.run(dataset, async ({ item, index }) => {
+  const response = await myAgent(item.input);
+  await evaluation.evaluate("ragas/answer_relevancy", {
+    index,
+    data: { input: item.input, output: response },
+    settings: { model: "openai/gpt-5-mini", max_tokens: 2048 },
+  });
+});
+```
+
+5. Run it. ALWAYS execute the experiment after creating it — an unrun experiment is useless. For Python notebooks: run the cells, or `jupyter nbconvert --to notebook --execute`. For TypeScript: `npx tsx experiment.ts`.
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 5; Style: 1/1/1

(MD029, ol-prefix)